plugins/node/opentelemetry-instrumentation-pino/src/instrumentation.ts

```diff
@@ -61,17 +60,22 @@ export class PinoInstrumentation extends InstrumentationBase {
                 args.splice(0, 0, {
                   mixin: instrumentation._getMixinFunction(),
                 });
-                return pinoModule(...(args as Parameters<Pino>));
+                return pinoModule(...(args as Parameters<any>));
```

Comment:
```suggestion
                return pinoModule(...args);
```
Not sure but I think that this cast is no loner needed after moving to `InstrumentationNodeModuleDefinition<any>`

Reply:
You're right: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1108/commits/e31c65f950d8ebecad92efcbd07c0da86167394f